### ISSUE TITLE
Could not find container for entity id [some id] - linkgraph.db with stale entries?

### ISSUE THREAD

**Issue by jacobat** [docker/docker#20229]
Every so often I see people reporting the error `Could not find container for entity id [some id]`. It appears to be related to the `--name` argument in the sense that `docker run --name foo ...` will show the error whereas `docker run ...` will not.

The solution is to remove `/var/lib/docker/linkgraph.rb`.

I think there's some edge case where docker doesn't cleanup properly. Or it could be related to unclean shutdowns of the vm's Docker runs in.

**Comment by jacobat** [comment]
`docker version`:

```
Client:
Version:      1.9.1
API version:  1.21
Go version:   go1.5.3
Git commit:   a34a1d5
Built:        Sun Jan 17 16:14:45 UTC 2016
OS/Arch:      darwin/amd64
Server:
Version:      1.9.1
API version:  1.21
Go version:   go1.4.2
Git commit:   a34a1d5
Built:        Fri Nov 20 13:12:04 UTC 2015
OS/Arch:      linux/amd64
```

`docker info`:

```
Containers: 33
Images: 286
Server Version: 1.9.1
Storage Driver: aufs
Root Dir: /var/lib/docker/aufs
Backing Filesystem: extfs
Dirs: 352
Dirperm1 Supported: false
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 3.13.0-24-generic
Operating System: Ubuntu 14.04 LTS
CPUs: 2
Total Memory: 3.861 GiB
Name: dockerhost
ID: YIPO:VJXT:DDE7:QK24:HSVN:WMUJ:DHS2:QIIW:Z3BE:CFVN:HSQC:Q5HS
```

Reproduction steps: Unknown.

**Comment by thaJeztah** [comment]
Thanks for reporting, however, this is a duplicate of https://github.com/docker/docker/issues/17691, and is resolved in docker 1.10 via https://github.com/docker/docker/pull/16032, which removes the dependency on SQLite (and the linkgraph.db)

I'll go ahead and close this issue, but feel free to comment here after I've closed
